[INFO -- 2017-03-08 06:38:23]
Starting flip_value script
version: 2.0
[INFO -- 2017-03-08 06:38:23]
Initialization strings:
[INFO -- 2017-03-08 06:38:23]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-03-08 06:38:23]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-08 06:38:23]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-08 06:38:23]
(4) file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-08 06:38:23]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-08 06:38:23]
(6) set args 228 2048 16 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_a_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_b_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/gold_2048_m-order_228_ths_16_blocks 1
[INFO -- 2017-03-08 06:38:23]
initSignal:28
[INFO -- 2017-03-08 06:38:23]
endSignal:28
[INFO -- 2017-03-08 06:38:23]
seqSignal:15
[INFO -- 2017-03-08 06:38:23]
faultMode:0
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 06:38:57]
pythonException: invalid frame
[INFO -- 2017-03-08 06:38:57]
Backtrace BEGIN:
#0  0x0000000000403709 in L_dgemm_185__par_region0_2_0 () at dgemm.c:221
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffe550 in ?? ()
#3  0x0000000000000000 in ?? ()

216	                            CC_arr(i,j) = 0.0;
217	
218	                    for (kg=kk,k=0; kg<MIN(kk+block,order); k++,kg++)
219	                        for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
220	                            for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
221	                                CC_arr(i,j) += AA_arr(i,k)*BB_arr(j,k);
222	
223	                    for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
224	                        for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
225	                            C_arr(ig,jg) += CC_arr(i,j);

Backtrace END
Memory content before bitflip:0x0a0x000x000x00
Fault Mode: Single bit-flip
base address to flip value: 0x7fffffffded8
address max offset: 4
Memory content after  bitflip:0x0a0x000x800x00
frame name: L_dgemm_185__par_region0_2_0
symbol name: j
symbol filename: dgemm.c
symbol line: 183
value: 8388618
value address: 0x7fffffffded8
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2017-03-08 06:38:57]
Fault Injection Successful after 34.638712883s
[INFO -- 2017-03-08 06:38:58]
event type: exit
[EXCEPTION -- 2017-03-08 06:38:58]
exit code: no exit code available
